sdk/servicebus/azure-servicebus/azure/servicebus/management/aio/_management_client_async.py

```diff
@@ -0,0 +1,259 @@
+         ServiceBus. Only a portion of properties can be updated.
+         Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue.
+        :type queue_description: ~azure.servicebus.management.QueueDescription
+        :returns: ~azure.servicebus.management.QueueDescription returned from ServiceBus.
+        """
+
+        if not isinstance(queue_description, QueueDescription):
+            raise TypeError("queue_description must be of type QueueDescription")
+
+        to_update = QueueDescription()
+
+        for attr in QUEUE_DESCRIPTION_SERIALIZE_ATTRIBUTES:
+            setattr(to_update, attr, getattr(queue_description, attr, None))
+        to_update.default_message_time_to_live = avoid_timedelta_overflow(to_update.default_message_time_to_live)
+        to_update.auto_delete_on_idle = avoid_timedelta_overflow(to_update.auto_delete_on_idle)
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_update,
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            try:
+                et = cast(
+                    ElementTree,
+                    await self._impl.queue.put(
+                    queue_description.queue_name,  # type: ignore
+                    request_body,
+                    api_version=constants.API_VERSION,
+                    if_match="*"
+                    )
+                )
+            except ValidationError:
+                # post-hoc try to give a somewhat-justifiable failure reason.
+                raise_with_traceback(
+                    ValueError,
+                    message="queue_description must be a QueueDescription with valid fields, "
+                            "including non-empty string queue name")
+
+        return _convert_xml_to_object(queue_description.queue_name, et, QueueDescription)  # type: ignore
+
+    async def delete_queue(self, queue_name):
+        # type: (str) -> None
+        """Delete a queue
+
+        :param str queue_name: The name of the queue
+        """
+
+        if not queue_name:
+            raise ValueError("queue_name must not be None or empty")
+        with _handle_response_error():
+            await self._impl.queue.delete(queue_name, api_version=constants.API_VERSION)
+
+    async def list_queues(self, skip=0, max_count=100):
+        # type: (int, int) -> List[QueueDescription]
+        """List the queues of a ServiceBus namespace
+
+        :param int skip: skip this number of queues
+        :param int max_count: return at most this number of queues if there are more than this number in
+         the ServiceBus namespace
+        """
+        return await self._list_queues(skip, max_count, QueueDescription)
+
+    async def list_queues_runtime_info(self, skip=0, max_count=100):
```

Comment:
skip and maxcount should be keyword only, and kwargs in the sync version. Same comment above for `list_queues`

Reply:
updated to use keyword